storybook/src/docs/developer-guide/using-links-with-routers.docs.mdx

```diff
@@ -0,0 +1,56 @@
+{/* @license CC0-1.0 */}
+
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Docs/Developer Guide/Using links with routers" />
+
+# Using links with routers
+
+The Amsterdam Design System provides various link components that render native anchor tags,
+typically without extra features.
+You may need to integrate these components into a routing solution for your application.
+The link components in the Amsterdam Design System are designed for easy extension.
+Below are some examples for common routing solutions:
+
+## Next.js
+
+You can hook into the Next router using the `legacyBehaviour` and `passHref` props on the Next link component, like so:
+
+```javascript
+import { Link } from "@amsterdam/design-system-react";
+import NextLink from "next/link";
+
+<NextLink legacyBehavior passHref href="/amsterdam">
+  <Link>Link label</Link>
+</NextLink>;
```

Comment:
Should we remove the semicolons for clarity, or is there a risk of errors after pasting?

Reply:
No, semicolons are generally useless 😁 Prettier added them automatically, I'll add an ignore line.